src/style.css

```diff
@@ -0,0 +1,146 @@
+*,
+*:after,
+*:before {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  background-color: tomato;
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+.wrap {
+  min-height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 20px;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+}
+
+.card__img {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+  display: block;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  margin-bottom: 24px;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.rating__stars {
+  display: flex;
+}
+
+.rating__star {
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.rating__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.rating__star:nth-child(1) {
+  background: url(images/star-active.svg) no-repeat;
+}
+
+.rating__star:nth-child(2) {
+  background: url(images/star-active.svg) no-repeat;
+}
+
+.rating__star:nth-child(3) {
+  background: url(images/star-active.svg) no-repeat;
+}
+
+.rating__star:nth-child(4) {
+  background: url(images/star-active.svg) no-repeat;
+}
+
+.rating__star:nth-child(5) {
+  background: url(images/star.svg) no-repeat;
+}
+
+.rating__review {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.price__item {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.price__uah {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  min-height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  color: #fff;
+  font-weight: 700;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  transition: all 0.3s ease 0s;
```

Comment:
bad practice to add a transition for `all` styles. Replace `all` with a style name that will change property values smoothly